lib/jazzy/source_declaration.rb

```diff
@@ -41,6 +41,13 @@ def fully_qualified_name
       namespace_path.map(&:name).join('.')
     end
 
+    # If this declaration is an objc category, returns an array with the name
+    # of the extended objc class and the category name itself, i.e.
+    # ["NSString", "MyMethods"], nil otherwise.
+    def objc_category_name
+      name.split(/[\(\)]/) if type.objc_category?
```

Comment:
Do you think I should split this method into two: one to return the category name and the other to return the extended class name? Or rename the above one to match the method signature?